src/JustEat.StatsD/PublicAPI.Shipped.txt

```diff
@@ -118,6 +118,7 @@ static JustEat.StatsD.StatsDServiceCollectionExtensions.AddStatsD(this Microsoft
 static JustEat.StatsD.TagsFormatter.CloudWatch.get -> JustEat.StatsD.TagsFormatters.IStatsDTagsFormatter!
 static JustEat.StatsD.TagsFormatter.DataDog.get -> JustEat.StatsD.TagsFormatters.IStatsDTagsFormatter!
 static JustEat.StatsD.TagsFormatter.InfluxDb.get -> JustEat.StatsD.TagsFormatters.IStatsDTagsFormatter!
+static JustEat.StatsD.TagsFormatter.GraphiteDb.get -> JustEat.StatsD.TagsFormatters.IStatsDTagsFormatter!
```

Comment:
It's fine to leave it here, but typically when adding new APIs they go into the `Unshipped` file and then get "promoted" to this file after release.

The main reason for that is that the analyser is stricter on things added to the Unshipped file for if, for example, a new overload would create ambiguity and potentially be a breaking change.